Mlem/Extensions/JSONDecoder+Default.swift

```diff
@@ -11,35 +11,34 @@ extension JSONDecoder {
     static var defaultDecoder: JSONDecoder {
         let decoder = JSONDecoder()
         decoder.keyDecodingStrategy = .convertFromSnakeCase
-
-        let formatter = DateFormatter()
-
-        formatter.timeZone = .gmt
-        formatter.locale = Locale(identifier: "en_US_POSIX")
-
+        
         let formats = [
+            "yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ",
             "yyyy-MM-dd'T'HH:mm:ss.SSSSSS",
-            "yyyy-MM-dd'T'HH:mm:ss.SSSSZ",
-            "yyyy-MM-dd'T'HH:mm:ss.SSSS",
-            "yyyy-MM-dd'T'HH:mm:ss",
-            "yyyy-MM-dd HH:mm:ss.SSSSSS",
-            "yyyy-MM-dd HH:mm:ss",
-            "yyyy-MM-dd"
+            "yyyy-MM-dd'T'HH:mm:ssZ",
+            "yyyy-MM-dd'T'HH:mm:ss"
         ]
+        
+        let formatters = formats.map { format in
```

Comment:
Not PR block worthy in this context since defaultDecoder isn't called often, but definitely should be careful recomputing expensive DateFormatter in get vars.